src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-errors.mdx

```diff
@@ -16,11 +16,17 @@ redirects:
         <TabsBarItem id="ios">
             iOS
         </TabsBarItem>
-        <TabsBarItem id="xamarin">
-            Xamarin
+        <TabsBarItem id="cordova">
+            Cordova
         </TabsBarItem>
-        <TabsBarItem id="maui">
-            .NET MAUI
+        <TabsBarItem id="capacitor">
+            Capacitor
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React native
```

Comment:
```suggestion
            React Native
```

Reply:
Also, is there any reason you can think of why the 'React Native' tab appears to be so much smaller than the other agent pages?

Reply:
I'm not sure, but that's a good catch. I'll look into it!